projects/ccd-case-ui-toolkit/src/lib/shared/components/search-result/search-result.component.html

```diff
@@ -35,12 +35,14 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseSta
       </th>
       <th *ngFor="let col of resultView.columns">
         <table class="search-result-column-header"
-          attr.aria-label="Sort by {{col.label}} {{ isSortAscending(col)? 'ascending' : 'descending' }}">
+          [attr.aria-label]="'Sort by ' + col.label + ' ' + isSortAscending(col)? 'ascending' : 'descending'">
```

Comment:
ascending and descending should be flipped here. If the sort is currently ascending, the label should say 'sort by decending', as that's what will happen when you click.

Reply:
This is a feature developed by someone else in the past (5fbc8735e4051f7a91e04b70e207a4d1100e5072). I just made the change to fix the sonar issues (aria-label wasn't showing properly).

Reply:
OK, I'll resolve for now, but please let @phillip-whitaker-hmcts know so he can address/follow up.